homeassistant/components/ozw/__init__.py

```diff
@@ -265,6 +267,22 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
     return True
 
 
+async def async_remove_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Remove a config entry."""
+    if not entry.data.get(CONF_INTEGRATION_CREATED_ADDON):
+        return
```

Comment:
Either this needs to be a boolean or the method return type is wrong.

Reply:
The typing needs updating.